src/utils/hints/accountHints.js

```diff
@@ -13,4 +13,9 @@ export const accountHints = {
   name: 'Name owned by this account.',
   expires: 'Keyblock height and estimated date and time when the name owned by the account expires.',
   pointsTo: 'The account that points to the name owned by the account.',
+  activityHash: 'Transaction hash of the transaction corresponding to the activity where the account was involved. Some of the activities are executed internally and therefore don\'t have a transaction hash.',
+  activityTime: 'Keyblock height and exact date and time when the activity has been executed by inclusion into a microblock.',
+  activityType: 'Type of the activity where the account was involved.',
+  activityMeaning: 'The meaning of the activity.',
+  activityData: 'The data displayed differs for each activity type.',
```

Comment:
here it's not easy for me to understand after reading it. Could you try something that would explain it for dummies? .)

Reply:
I literally copied it from the account transaction's time hint above :D I would keep it this way for consistency or improve it everywhere (but I don't have a good idea how)

Reply:
Ok, as you wish. For me was not easy to understand